assets/js/admin.js

```diff
@@ -103,6 +106,34 @@ jQuery(document).ready(function($) {
         }, 'json');
     } );
 
+    /*
+    Initial setup
+     */
+    $('.js-a0-setup-input').keydown(function(e){
+        if(13 === e.keyCode) {
+            e.preventDefault();
+            return false;
+        }
+    });
+
+    $('.js-a0-select-setup').click(function (e) {
+        e.preventDefault();
+        $('#profile-type').val($(this).attr('data-profile-type'));
+        $('#connectionSelectedModal').modal();
+    });
+
+    $('#manuallySetToken').click(function (e) {
+        e.preventDefault();
+        $('#enterTokenModal').modal();
+        $('#connectionSelectedModal').modal('hide');
+        return false;
```

Comment:
On some languages at least, this boolean that you return is for "The event has already been handled". If that's the case, why are some returning it and some others not?